robot/Cumulus/tests/browser/zpayment_allocations/enable_pmt_alloc_and_test.robot

```diff
@@ -18,11 +18,10 @@ Suite Teardown  Run keywords
 *** Variables ***
 &{contact1_fields}         Email=test@example.com
 &{contact2_fields}         Email=test@example.com
```

Comment:
Same comment as above.

Reply:
actually there is no use of email here just giving as some data other than just name.